test/e2e/helpers.go

```diff
@@ -973,6 +973,16 @@ func InstallHelmChart(ctx context.Context, clusterProxy framework.ClusterProxy,
 			chartRequested, err := helmLoader.Load(cp)
 			g.Expect(err).NotTo(HaveOccurred())
 			release, err := i.RunWithContext(ctx, chartRequested, vals)
+			if err != nil {
+				Logf("Failed to install release %s, attempting to cleanup so we can retry", releaseName)
+				// Best effort attempt to delete the failed release so we can retry.
+				_, delErr := helmAction.NewUninstall(actionConfig).Run(releaseName)
+				if delErr != nil {
```

Comment:
ditto here about Gomega, I think a simple `g.Expect(delErr).NotTo(HaveOccurred())` will give us the equivalent amount of specific messaging that manually logging an error does, with additional data about what's in the `delErr`